src/app/features/charts/components/score-chart/score-chart.component.ts

```diff
@@ -30,10 +30,12 @@ export class ScoreChartComponent extends BaseChartComponent {
     this.init();
   }
 
+  @Input() barColor = '#8b8ad1';
```

Comment:
Going forward, more CSS props will be passed in.  In this case, we are using this component for each score chart, so we have a default color, but a color can be passed in to override the default color.